tests.js

```diff
@@ -0,0 +1,71 @@
+// change to true if you want the test to run
+// results will be displayed in the console
+export const RUN_TESTS = true;
+
+const RGBtestCases = [
+  // [Case, Value, Expected result]
+
+  ["HEX", "#FFFFFF", true],
+  ["HEX no hash", "FFFFFF", false],
+  ["HEX invalid", "#FFFFFX", false],
+  ["HEX too long", "#FFFFFFF", false],
+  ["HEX too short", "#FFFF", false],
+
+  ["HEX3", "#FFFFFF", true],
+  ["HEX3 invalid", "#FFX", false],
+  ["HEX3 no hash", "FFF", false],
+  ["HEX3 too short", "#FF", false],
+
+  // add more test cases
+];
+
+const contrastRatioTestCases = [
```

Comment:
if we are going to use it, more test cases required.

At least one for every possible format pair. The more - the better.

Reply:
When all inputs are converted to RGB format before calculation, it does not matter what format is given. They could all be in HEX.
Example:
Test case: `["hsl(240, 100%, 50%)", "#AAAB1C", 3.5]`, the two colours are first converted to RGB, then used in the calculation. So it does not matter if you write it like that or this `["rgb(33, 33, 222)", "rgb(170, 171, 28)", 3.5]`.